index.js

```diff
@@ -1,11 +1,12 @@
 addEventListener('fetch', event => {
   event.respondWith(handleRequest(event.request))
 })
-
 /**
- * Fetch and log a request
+ * Respond with hello worker text
  * @param {Request} request
  */
 async function handleRequest(request) {
-  return new Response('Hello worker!', { status: 200 })
+  return new Response('Hello worker!', {
+    headers: { 'content-type': 'text/plain' },
```

Comment:
status 200 is the default, so I found that irrelevant. I wanted to include one example of modifying `ResponseInit` though and it's best praactice to include `Content-type` anyways so I added that